app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt

```diff
@@ -28,138 +32,169 @@ class OverdueListAdapter @Inject constructor() : ListAdapter<OverdueListItem, Ov
 
```

Comment:
There are some IDE inspection warnings in this class. Can you either suppress them or fix them?

Reply:
Made functions `private`. Let me know if there are others.